json-endpoints/class.wpcom-json-api-update-post-endpoint.php

```diff
@@ -519,7 +528,12 @@ function write_post( $path, $blog_id, $post_id ) {
 
 				// Custom meta description can only be set on sites that have a business subscription.
 				if ( Jetpack_SEO_Posts::DESCRIPTION_META_KEY == $meta->key && ! Jetpack_SEO_Utils::is_enabled_jetpack_seo() ) {
-					return new WP_Error( 'unauthorized', __( 'SEO tools are not enabled for this site.', 'jetpack' ), 403 );
+					return new WP_Error( 'unauthorized', __( 'SEO Tools are not enabled for this site.', 'jetpack' ), 403 );
+				}
+				if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
```

Comment:
I included both conditionals here -- the prior was from Jetpack, the latter from WPCOM.  Do we need both?